.travis.yml

```diff
@@ -7,6 +7,9 @@ language: php
 php:
     - 7.1
     - 7.2
+    - 7.3
+    - 7.4
+    - nightly
```

Comment:
I wouldn't add nightly here, or at least define it as allowed failure?

Reply:
I think `nightly` means php-8.0.0-RCx for the moment because it's not a stable version yet. We can add `- 8.0` once it's available in Travis and define `nightly` as allowed failures then. What do you think?